echo/echo.go

```diff
@@ -0,0 +1,86 @@
+package echo
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/vuln-list-update/utils"
+)
+
+const (
+	echoDir            = "echo"
+	advisoriesURLBase  = "https://advisory.echohq.com"
+	advisoriesFilePath = "data.json"
+)
+
+type option func(c *Updater)
+
+func WithVulnListDir(v string) option {
+	return func(c *Updater) { c.vulnListDir = v }
+}
+
+func WithBaseURL(v *url.URL) option {
+	return func(c *Updater) { c.baseURL = v }
+}
+
+func WithFilePath(v string) option {
+	return func(c *Updater) { c.filePath = v }
+}
```

Comment:
Do you need this. 
In test you can just return required file in `HandlerFunc`

Reply:
You're right, I removed it